examples/getSourceSet.py

```diff
@@ -353,8 +355,8 @@ def writeSourceSet(sourceSet, outfile="-"):
         fd = open(outfile, "w")
 
     for s in sourceSet:
-        print >> fd, s.getId(), s.getXAstrom(), s.getYAstrom(), s.getRa(
-        ), s.getDec(), s.getPsfFlux(), s.getFlagForDetection()
+        print(s.getId(), s.getXAstrom(), s.getYAstrom(), s.getRa(
+        ), s.getDec(), s.getPsfFlux(), s.getFlagForDetection(), file=fd)
```

Comment:
to my eyes, the line break here is weird. I'd expect the new line to occur at `s.getRa(),`